src/graphic/Fast3D/gfx_pc.cpp

```diff
@@ -3604,44 +3614,70 @@ const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> otrHandlers = {
     { G_SETINTENSITY, gfx_set_intensity_handler_custom },            // G_SETINTENSITY (0x40)
 };
 
-const static std::unordered_map<uint32_t, GfxOpcodeHandlerFunc> f3dex2Handlers = {
+#ifdef F3DEX_GBI_2
```

Comment:
Why ifdef the handler array? We should be switch arrays based on the renderer state current GBI value.